libs/dh/shared/domain/src/lib/generated/v1/model/search-criteria.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * DataHub BFF
+ * Backend-for-frontend for DataHub
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+export interface SearchCriteria { 
```

Comment:
Suggestion: Could we prefix these types as search-criteria etc. is pretty generic to avoid possible conflicts in the future?